docs/source/executing-operations/defer-support.mdx

```diff
@@ -40,6 +40,19 @@ To learn the basics of the `@defer` directive and how you can use it with your s
 
 The remainder of this article covers the Apollo Router's defer implementation in greater depth.
 
+## Executing a `@defer` query
+
+To execute a deferred query on the Apollo Router, a GraphQL client sends an HTTP request with _almost_ the exact same format that it uses for usual query and mutation requests.
+
+The only difference is that the request should include the following `Accept` header:
+
+```text title="Example header"
+Accept: multipart/mixed; deferSpec=20220824, application/json
```

Comment:
Nit: [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) uses space after `,` but no space after `;`:

```suggestion
Accept: multipart/mixed;deferSpec=20220824, application/json
```

I kind of like it because it's easier to "see" the different list values. But either works with me.